src/lib/server/cms.ts

```diff
@@ -40,6 +132,12 @@ type TokenObject =
 			display: string | undefined;
 			raw: string;
 	  }
+	| {
+			type: 'externalProductWidget';
```

Comment:
Why is the `externalProductWidget` not the same as the regular product widget (with a URL that just so happens to be remote)? Why do we care about the fact it is remote?